tests/test_encodings.py

```diff
@@ -420,10 +421,43 @@ def test_mds_float64(self, decoded: float, encoded: bytes):
         assert isinstance(dec, np.floating)
         assert dec == decoded
 
+    @pytest.mark.parametrize(('decoded', 'encoded'), [(42, b'42')])
+    def test_mds_StrInt(self, decoded: int, encoded: bytes):
+        coder = mdsEnc.StrInt()
+        enc = coder.encode(decoded)
+        assert isinstance(enc, bytes)
+        assert enc == encoded
+
+        dec = coder.decode(encoded)
+        assert isinstance(dec, int)
+        assert dec == decoded
+
+    @pytest.mark.parametrize(('decoded', 'encoded'), [(42.0, b'42.0')])
+    def test_mds_StrFloat(self, decoded: float, encoded: bytes):
```

Comment:
What would happen when it is a negative float ? Can you please also add the test case for the same?